.eslintrc.json

```diff
@@ -33,6 +33,19 @@
         "@typescript-eslint/no-empty-function": "off",
         "@typescript-eslint/explicit-function-return-type": "off",
         "@typescript-eslint/camelcase": "off",
+        "@typescript-eslint/naming-convention": [
+          "warn",
+          {
+            "selector": "variable",
+            "format": ["camelCase", "UPPER_CASE"],
+            "leadingUnderscore": "allow",
+            "trailingUnderscore": "allow",
+            "custom": {
+              "regex": "any|[Nn]umber|[Ss]tring|[Bb]oolean|[Uu]ndefined",
```

Comment:
I'm a little worried about this list of exceptions growing unbounded, is there a configuration setting that we could use that just captures common constants?

Reply:
I was not a fan of this either, but wanted parity with the old `variable-name` rule.

The scope of this regex is reserved words. I looked, but was not able to find a configuration that had equivalency, or a dictionary of such words that could be used in this context.

This list was taken from a list of banned keywords in tslint's variable-name feature: https://github.com/palantir/tslint/blob/a66f3b3430b4d3f03ff93fcef94609b88809d4e1/src/rules/variableNameRule.ts#L26-L36

Reply:
@bcoe I have remove the keyword blacklist regex